task/04-arrays-tasks.js

```diff
@@ -249,7 +255,7 @@ function getMovingSum(arr) {
  * [ "a" ] => []
  */
 function getSecondItems(arr) {
-   throw new Error('Not implemented');
+   return arr.filter(x => arr.indexOf(x) % 2 !== 0);
```

Comment:
Use more meaningful names for variables.

Reply:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.